contracts/avalanche/connectors/Qi/helpers.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import { DSMath } from "../../common/math.sol";
+import { Basic } from "../../common/basic.sol";
+import { ComptrollerInterface, QiInterface, BenqiMappingInterface } from "./interface.sol";
+
+abstract contract Helpers is DSMath, Basic {
+    /**
+     * @dev Benqi Comptroller
+     */
+    ComptrollerInterface internal constant troller = ComptrollerInterface(0x486Af39519B4Dc9a7fCcd318217352830E8AD9b4);
+
+    /**
+     * @dev Reward Token
+     */
+    QiInterface internal constant benqiToken = QiInterface(0x8729438EB15e2C8B576fCc6AeCdA6A148776C0F5);
+
+    /**
+     * @dev Benqi Mapping
+     */
+    BenqiMappingInterface internal constant qiMapping = BenqiMappingInterface(0xe7a85d0adDB972A4f0A4e57B698B37f171519e88);
```

Comment:
TODO: Need to change this after the deployment of `InstaBenqiMappingAvalanche` contract

Reply:
@babkendev01 Awesome! Looks good! Would you be able to deploy `InstaBenqiMappingAvalanche` contract? With all the markets? After that, we can update `InstaBenqiMappingAvalanche` contract address